File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.java
class Solution {
    public String solution(String my_string, int s, int e) {
        String answer = "";
        char[] arr = my_string.toCharArray();
        char[] tempArr = my_string.toCharArray();
        
        for(int i = 0; i < e - s + 1; i++){            
            arr[s + i] = tempArr[e - i];
        }
    
        return answer = String.valueOf(arr);
    }
}

File: 프로그래머스/0/120822. 문자열 뒤집기/문자열 뒤집기.py
def solution(my_string):
    answer = ''
    i = -1
    while i >= -(len(my_string)) :
        if i == -1 :
             answer += my_string[i : ]
             #print(my_string[i : ])
        else : 
             answer += my_string[i : i + 1]                
        #print(my_string[i : i + 1])
        #print(answer)
        i += -1
        
    return answer